### DISCUSSION TITLE
Update buton

### DISCUSSION THREAD

**Discussion by Gunter1710** [jomjol/AI-on-the-edge-device#649] [Ideas] [answered]
Hi, 
the possibility of carrying out an OTA update is really great. It would be nice if you could run the update directly, i.e. without downloading it from github beforehand.

**Comment by jomjol** [comment, marked as answer]
Well, most of the users don't want to have acces of the device to the internet directly. Therefore this is not implemented. Would be an option for a professional version.

**Reply by caco3** [reply]
software or devices which call home (call github) unwanted usually are something most people do not want.
There where some changes to the OTA page, we added now a direct link to the release page.
Also we could add a button which would do the check on explicit call, but somebody would have to integrate that (I would do it on javascript level).
